hoomd/md/update.py

```diff
@@ -203,7 +205,8 @@ def __init__(self, group, r=None, rx=None, ry=None, rz=None, P=(0,0,0)):
         self.rz = rz
         self.metadata_fields = ['group','P', 'rx', 'ry', 'rz']
 
-class mueller_plathe_flow(_updater):
+
+class MuellerPlatheFlow(Updater):
     R""" Updater class for a shear flow according
```

Comment:
```suggestion
    """Mueller/Plathe method to establish shear flow.
```

Reply:
This is actually one person with a hyphenated name: "Müller-Plathe" or "Mueller-Plathe"

Reply:
> This is actually one person with a hyphenated name: "Müller-Plathe" or "Mueller-Plathe"

Thanks, I edited my suggestion accordingly. 

Perhaps we should also include the umlaut in the class name for correctness?

Reply:
I'm not a fan of unicode characters in class names, but maybe I am old fashioned. I think both spellings are acceptable, so I'm OK either way. This method can also be called RNE ("reverse nonequilibrium") or reverse perturbation, so `RNEFlow` and `ReversePerturbationFlow` would also be good names.

Reply:
Sounds good. Let's call it `ReversePerturbationFlow`. @danevans09 can you rename the class at the Python level? No need to make C++ modifications at this time, we can defer that until the proposed later refactoring of this class.